FND/Accounting/CurrencyAmount.rdf

```diff
@@ -366,7 +366,19 @@ The definition of currency provided herein is compliant with the definitions giv
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>unit of account identifier</rdfs:label>
-		<skos:definition>sequence of characters that can be used to uniquely identify a unit of account</skos:definition>
+		<skos:definition>sequence of characters that can be used to uniquely identify a nominal monetary unit</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-acc-cur;UnitPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;isPriceFor"/>
+				<owl:someValuesFrom rdf:resource="&owl;Thing"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>unit price</rdfs:label>
+		<skos:definition>monetary price expressed in relation to a well-known measure such as quantity, weight, volume, duration or other measurable unit by which the goods or services are allocated</skos:definition>
```

Comment:
@rivettp I think the range of isPriceFor, in most cases, such as for a commodity, would be Quantity or possibly QuantityValue, which would provide the specific unit.  The definition came from the retail DTF, and motivation was specifically for commodities in addition to shares.  We have a gap there that we need to address for the CFTC, this was just the start for addressing that.  We do need to either use an existing units ontology or add specific units for the commodities in question - I didn't get that far under this issue resolution, as I was hoping to at least add price per share, which you commented on, below.  

MonetaryPrice does inherit from quantity value, but hasAmount is a subproperty of hasNumericValue, and there is an exactly 1 restriction on both, so there could be some odd reasoning results if someone uses both.  We should revisit that - I think there are gremlins lurking so creating an example or two would be good.  The gremlins would be due to subclassing quantity value, and/or having hasAmount being a subproperty of hasNumericValue, so I think we still have something to fix. 

The idea with respect to share price is that this should be the general case, so that we can use it with commodities.  We should try a couple of examples, maybe in DER next week if I don't get to it before then.  If you have a better suggestion for a definition, or thoughts on how to revise this one, please suggest something.

Reply:
I'm confused by this change; I normally use "price" as an example of how the obvious relationships aren't always what you want in a model.  An equity doesn't directly have a price; an equity has an offer (and sometimes a bid), and those things have prices.  Actually (per @rivettp's comment), the equity doesn't have a price; a particular amount of that equity has a price, depending on how the equity is measured; e.g., a massage is measured by the hour, pork belly by the kilo, eggs by the dozen (or other count), etc.  

Also, I can't exactly see the change that was made; something went wrong with github's diff, and the diffs show several apparently identical lines as a diff.  I'll have to check it out and do a local diff to see it.  Maybe the change doesn't impact the bid/offer structure, but I can't really tell. 

By the way, Good Relations does a good job of organizing this; they have a thing called a TypeAndQuantityNode (combining amount and the type of Good or Service), and the amount has units on it as well.  Then the offer brings together a price with the TyyeAndQuantityNode.  I think FIBO is actually isomorphic to this (I'd have to track a bit to figure this out); I don't  understand why we need more.

Reply:
@dallemang I'll take a look at good relations, although the concept we were missing was unit price, regardless of whether that applies to an offer or something else.  Price per share was also missing - even to track the current market price if someone needs that.

In any case, I sent email on the issue I'm having with respect to github, which came up when several security updates in source tree required me to download a local git client (recommended by Atlassian's community).  I'm not sure what feature has changed, and it appears to be applied after the serializer runs but before a change is fully committed, and changes the newline/CRLF/LF on every line (which is why the diffs aren't working properly).  If you have suggestions as to what I need to tweak in my configuration to fix that, I would really appreciate it.  Pete suggested I add a line to my config file for 'autolf=true', which I did, but I'm not sure that's doing anything, or if it is, perhaps it's not the right thing.